File: Prod.RutaDigital.Entidades/CapacitacionResultado/CapacitacionResultado.cs
namespace Prod.RutaDigital.Entidades;

public class CapacitacionResultado : Auditoria
{
    public int id_capacitacion_resultado { get; set; }
    public int id_resultado { get; set; }
    public int id_recomendacion { get; set; }
    public DateTime? fecha_inicio { get; set; }
    public DateTime? fecha_fin { get; set; }
    public decimal progreso { get; set; }
    public bool concluido { get; set; }
    public int calificacion { get; set; }
}

File: Prod.RutaDigital.Presentacion.Angular/ClientApp/src/app/recomendaciones/interfaces/capacitacion-resultado.ts
export interface CapacitacionResultado {
  id_capacitacion_resultado: number;
  id_resultado: number;
  id_recomendacion: number;
  fecha_inicio: Date;
  fecha_fin: Date;
  progreso: number;
  concluido: boolean;
  clasificacion: number;

  id_modulo: number;
  orden_modulo: number;
  nombre_modulo: string;

  id_nivel_madurez: number;
  nombre_nivel_madurez: string;

  orden_recomendacion: number;
  descripcion_recomendacion: string;
  imagen_recomendacion: string;
}
